### PR TITLE
[SUCE-194] Suttaplex adjustments

### PR DESCRIPTION
**Pull request by gradam**

### CODE REVIEW

File: server/server/data_loader/dictionaries.py [hunk 1]
```diff
@@ -12,10 +12,13 @@ def load_dictionaries(db, dictionaries_dir):
...
-            })
+            try:
+                from_lang, to_lang = dictionary.stem.split('2')
+                docs.append({
+                    'from': from_lang,
+                    'to': to_lang,
+                    'dictionary': json.load(f)
+                })
+            except ValueError:
+                pass
```
Comment (TrebuhD) [inline server/server/data_loader/dictionaries.py]: I thought this was temporary?

File: server/server/data_loader/textdata.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def uids_are_related(uid1, uid2, _rex=regex.compile(r'\p{alpha}*(?:-\d+)?')):
     def is_bold(self, lang, element):
         if element.tag in {'b', 'strong'}:
             return True
-        if lang in {'zh', 'lzh', 'ko', 'jp', 'tw'}:
+        if lang in {'lzh', 'lzh', 'ko', 'jp', 'tw'}:
```
Comment (TrebuhD) [inline server/server/data_loader/textdata.py]: two `lzh`s?

File: server/server/data_loader/dictionaries.py (hunk 1 above)
Comment (gradam) [reply server/server/data_loader/dictionaries.py]: Exactly. Till the way we are supposed to hold dictionaries will be clear so it has to stay here for now

Comment (TrebuhD) [reply server/server/data_loader/dictionaries.py]: Maybe at least print the `Exception`?

Comment (gradam) [reply server/server/data_loader/dictionaries.py]: We could. A will add something

**Pull request merged**